auth-web/src/views/auth/UnauthorizedView.vue

```diff
@@ -52,7 +52,7 @@ export default class UnauthorizedView extends Vue {
 
   beforeMount () {
     // TODO: Replace the login source check with whatever value for director search account.
-    this.isDirSearchUser = (this.currentUser?.loginSource === 'BCSC')
+    this.isDirSearchUser = (this.currentUser?.loginSource === 'BCROS')
```

Comment:
may be can we use a constant  for BCROS and use it across?

Reply:
We should extract the login source to a enum, been meaning to do this

Reply:
I agree, will do that way